content/blog/2022/01/camel-k-release-1-8/index.md

```diff
@@ -22,26 +22,30 @@ As usual, thanks to Apache Camel, Camel Quarkus and Kamelet Catalog contributors
+We had this feature in our TODO list since a while. You will be able to use your HTTP(S) proxy in the Camel K Operator for any kind of egress operation (Maven dependencies, image pulling, ...). Moreover, once set, you will be able to have it automatically **configure the HTTP proxy settings** in all your `Integrations`. Just declare the typical HTTP proxy environment variables during Camel K Operator installation. Have a look a the [official documentation](/camel-k/next/configuration/http-proxy.html) for more details. 
 
 ## Configurable Maven CLI options
 
 Maven is our preferred tool for managing projects dependencies. However, sometimes it is very verbose, in particular when it downloads the Internet... Jokes apart, we introduced the possibility to set the `.IntegrationPlatform.spec.build.maven.cliOption`, which will allow you **provide any `maven` configuration you want** (ie, `-V,--no-transfer-progress,-Dstyle.color=never, ...`). Your log aggregation tool will breath now.
 
-## Encapsulated configuration, volumens and properties logic into traits
+## Encapsulated configuration, volumes and properties logic into traits
 
 This is a bit an hidden feature, but you must be aware of what's behind the scenes. We moved part of the logic previously run by the `kamel run` command, into a dedicated trait, which will be in charge to take care of that aspect only. With this strategy we aim to **simplify the `Integration` configuration** and the execution which is not done via `kamel` CLI.
 
 More in detail, with this release we've moved the `-p|--property` logic into the [Camel trait](/camel-k/next/traits/camel.html). We've also created the [Mount trait](/camel-k/next/traits/mount.html) which will be take care of `--config`, `--resource` and `--volume` parameters of the `kamel run` CLI command. Nothing will change for the CLI users. If you instead are directly editing the `Integration`, be aware that we have deprecated the `.Integration.Configuration` and `Integration.Resources` field in favour of those traits. We will still support it for a few more releases, but you should no longer use it in the future.
 
+## Report runtime health checks into Integration readiness condition
+
+In the latest releases we're improving the monitoring aspect in order to provide the user all the information about the health of an `Integration`. In this release we're adding a **readiness condition** which will tell you more about the healthy status of an Integration. Also here, we're taking the opportunity of this work to encapsulate the logic into dedicated traits (`health` trait in this case). As a side work, we've also decoupled liveness and readiness probes (`container` trait).
```

Comment:
* `adding a **readiness condition** which will tell you more about the healthy status of an Integration` -> `improving the **readiness condition** which now incorporates the Camel runtime health status into the integration status
* `Also here, we're taking the opportunity of this work to encapsulate the logic into dedicated traits`: This has been done in the previous release already.

Reply:
`As a side work, we've also decoupled liveness and readiness probes (container trait).` can also be removed, as it's been part of 1.7.0 already.